packages/p2p/src/components/orders/order-details/order-details.jsx

```diff
@@ -0,0 +1,194 @@
+import React         from 'react';
+import { Button }    from 'deriv-components';
+import { localize }  from 'deriv-translations';
+import FooterActions from 'Components/footer-actions/footer-actions.jsx';
+import './order-details.scss';
+
+const OrderInfoBlock = ({ label, value }) => (
+    <div className='order-details__info-block'>
+        <p className='order-details__info-block-label'>{ label }</p>
+        <strong className='order-details__info-block-value'>{ value }</strong>
+    </div>
+);
+
+const OrderDetailsStatusBlock = ({ order_details }) => {
+    const {
+        is_buyer,
+        is_buyer_cancelled,
+        is_buyer_confirmed,
+        is_expired,
+        is_pending,
+        is_seller_confirmed,
+    } = order_details;
+
+    return status_to_show ? (
+        <h2 className='order-details__header-status'>
+            { is_pending && is_buyer &&
```

Comment:
can we sort this for is_buyer and !is_buyer

Reply:
@mustofa-binary We tried that, it doesn't look good, and there are a couple of cases where it doesn't matter if the user is buyer or seller. This is more readable.